lib/kubernetes-deploy/statsd.rb

```diff
@@ -4,37 +4,14 @@
 
 module KubernetesDeploy
   class StatsD
-    extend ::StatsD
-
     PREFIX = "KubernetesDeploy"
 
     def self.duration(start_time)
       (Time.now.utc - start_time).round(1)
     end
 
-    def self.build
-      if ENV['STATSD_DEV'].present?
-        self.backend = ::StatsD::Instrument::Backends::LoggerBackend.new(Logger.new($stderr))
-      elsif ENV['STATSD_ADDR'].present?
-        statsd_impl = ENV['STATSD_IMPLEMENTATION'].present? ? ENV['STATSD_IMPLEMENTATION'] : "datadog"
-        self.backend = ::StatsD::Instrument::Backends::UDPBackend.new(ENV['STATSD_ADDR'], statsd_impl)
-      else
-        self.backend = ::StatsD::Instrument::Backends::NullBackend.new
-      end
-    end
```

Comment:
I *think* the defaults that we pick up from the environment are fine in this case. The new client uses `datadog` as default implementation.